src/paperless/settings.py

```diff
@@ -351,11 +351,23 @@ def _parse_beat_schedule() -> dict:
 
 ROOT_URLCONF = "paperless.urls"
 
-FORCE_SCRIPT_NAME = os.getenv("PAPERLESS_FORCE_SCRIPT_NAME")
-BASE_URL = (FORCE_SCRIPT_NAME or "") + "/"
-LOGIN_URL = BASE_URL + "accounts/login/"
-LOGIN_REDIRECT_URL = "/dashboard"
-LOGOUT_REDIRECT_URL = os.getenv("PAPERLESS_LOGOUT_REDIRECT_URL")
+
+def _parse_base_paths() -> tuple[str, str, str, str, str]:
+    script_name = os.getenv("PAPERLESS_FORCE_SCRIPT_NAME")
+    base_url = (script_name or "") + "/"
+    login_url = base_url + "accounts/login/"
+    login_redirect_url = base_url + "dashboard"
+    logout_redirect_url = os.getenv("PAPERLESS_LOGOUT_REDIRECT_URL", base_url)
+    return script_name, base_url, login_url, login_redirect_url, logout_redirect_url
+
+
+base_paths = _parse_base_paths()
```

Comment:
Just a suggestion, you could do something like
`FORCE_SCRIPT_NAME, BASE_URL, ... = _parse_base_paths()`
to avoid creating the temporary `base_paths`

Reply:
Oh yea good point, Im more used to that in JS, forgot about it in Python =)
Thanks